### ISSUE TITLE
Select Component - label is orphaned from element in WAVE tool

### ISSUE THREAD

**Issue by adamshafer** [primefaces/primevue#7961]
### Describe the bug

In the latest version of PrimeVue, there is no proper way to attach a <label> element to the <Select> component. You'll see in the docs that the WAVE (web accessibility evaluation tool) that these labels are being orphaned, even when using the <label for="xyz" />XYZ</label><Select inputId="xyz" /> format as shown in the demo:

<img width="3978" height="2330" alt="Image" src="https://github.com/user-attachments/assets/276c556b-c542-450f-8daa-91a128a890d0" />

### Reason for not contributing a PR

- [x] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/primevue-4-vite-issue-template-5wryvgrx?file=src%2FApp.vue

### Environment

.

### Vue version

3.5.18

### PrimeVue version

4.3.6

### Node version

22.16.0

### Steps to reproduce the behavior

Create any select element as shown with a proper label. Observe that the two are not connected and report "orphaned form label" in accessibility testing tools.

### Expected behavior

I'd expect these not to throw "orphaned form label" errors in accessibility testing.

**Comment by c0lt** [comment]
We're using Storybook with Accessibility tests and we also got the error that there's no label. We solved it by giving the `<label>` element an `id` and setting the `ariaLabelledby` prop.

**Comment by adamshafer** [comment]
I also started going this route, but we've got 232 instances (and counting)
of the <Select /> element in our app and I'd rather not change every one. :)

**Comment by gwalter98** [comment]
> We're using Storybook with Accessibility tests and we also got the error that there's no label. We solved it by giving the `<label>` element an `id` and setting the `ariaLabelledby` prop.

I also noticed this issue through Storybook's Accessibility add-on. Using PrimeVue's doucmentation for [Float Label with Select](https://primevue.org/select/#floatlabel) I get the following error: _ARIA input fields must have an accessible name_.

It seems that because the underlying DOM element for the Select component is a `<span>`, even with an `inputId` attribute set to the same value as the `for` attribute of the `label` the component is failing the accessibility tests as I think `for` only supports native input elements.

Also it appears that, despite being used for an example in the documentation, the `inputId` prop has been deprecated. There is a `labelId ` prop but this seems to just give the underlying `<span>` element an `id` of the given value. I wonder if this is intentional?

The above solution provided by @c0lt does work but should the component not be accessible by default? Nevertheless, I'm finding the documentation to be a little misleading for this scenario.

**Comment by IdaOstlund** [comment]
I think the issue is when the select is not editable. In that case it is rendered as a span and not an input as when it is editable. At least it seems to work for the editable version when adding labelId="myLabelId" and setting for="myLabelId" on the label. 

Adding aria-labelledby however seems to break things for the editable version and for me the not editable version still does not get the correct label read out, using VoiceOver. The best result I have managed to achieve is when using aria-label. The editable version would then read what I expect, the not editable version reads "space, My Label, ....." (when no option selected). 

Also, although it is maybe a slightly different matter, things like required and aria-describedby is not added to the correct element, only to root. When using pt I can manage to get aria-describedby to be read out with VoiceOver, but required only works for the editable version.

**Comment by zcdunn** [comment]
This sounds related to https://github.com/primefaces/primevue/issues/2924

**Comment by clemsontiger** [comment]
@IdaOstlund could you provide an example of how you're using aria-label here for when the select is rendered as a span? tried  solution provided by @c0lt bur WAVE is still flagging it.

**Comment by IdaOstlund** [comment]
@clemsontiger I did not manage to make it work perfectly, wave is still giving an "Orphaned form label" alert for when it is rendered as a span.  But what I did was similar to this:

```
<FloatLabel>
            <Select  
            labelId="over_label" 
            aria-label="Over Label" 

            v-model="value1"
            :options="cities" 
            optionLabel="name" />

            <label for="over_label">Over Label</label>
</FloatLabel>
```

**Comment by tugcekucukoglu** [comment]
This issue will be fixed in version 5.0.2. Thank you.
